CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing code or documentation
```

Comment:
I have consolidated the changes here, combined with other changes in my recent PR, so feel free to remove it here to avoid conflict

Reply:
Thank you, I will merge this PR soon!